examples/gnrc_border_router/Makefile.usb-cdc-ecm.conf

```
@@ -0,0 +1,12 @@
+# USB Modules
+USEMODULE += auto_init_usbus
+USEMODULE += usbus_cdc_ecm
+
+ifeq (1,$(USE_DHCPV6))
+  FLAGS_EXTRAS += --use-dhcpv6
+endif
+
+# Configure terminal parameters for UHCP
+  TERMDEPS += host-tools  
+  TERMPROG ?= sudo sh $(RIOTTOOLS)/usb-cdc-ecm/start_network.sh
+  TERMFLAGS ?= $(FLAGS_EXTRAS) $(IPV6_PREFIX) $(PORT)
```

```suggestion
TERMDEPS += host-tools  
TERMPROG ?= sudo sh $(RIOTTOOLS)/usb-cdc-ecm/start_network.sh
TERMFLAGS ?= $(FLAGS_EXTRAS) $(IPV6_PREFIX) $(PORT)
```

no need to indent those

---

73d295f9f